app/cinema/hall.py

```diff
@@ -0,0 +1,18 @@
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self, movie_name: str,

```

Comment:
Put all argument to a new line pleas
```suggestion
            self, 
            movie_name: str,

```